src/Query/QueryConditionsInterface.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Query;
+
+use Yiisoft\Db\Exception\NotSupportedException;
+use Yiisoft\Db\Expression\ExpressionInterface;
+
+interface QueryConditionsInterface
```

Comment:
QuryConditions - incorrect name for select, from union and etc.
QueryBase ?
And how about splitting to other folders with QueryBuilder's classes?

Reply:
rename for `QueryParts::class`.